src/Symfony/Component/Security/Core/Exception/AuthenticationExpiredException.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\Security\Core\Exception;
 
 /**
- * AuthenticationServiceException is thrown when an authenticated token becomes un-authentcated between requests.
+ * AuthenticationServiceException is thrown when an authenticated token becomes un-authenticated between requests.
```

Comment:
Is this about `AuthenticationExpiredException` anyway? message states AuthenticationServiceException which by itself has a different doc.